app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comments_counter` should be marked as private after implementing the `after_save` method.